dev/MRTCore/mrt/Core/src/MRM.cpp

```diff
@@ -25,6 +25,7 @@ typedef struct
 
 constexpr wchar_t ResourceUriPrefix[] = L"ms-resource://";
 constexpr int ResourceUriPrefixLength = ARRAYSIZE(ResourceUriPrefix) - 1;
+constexpr wchar_t c_defaultResourceFilename[] = L"resources.pri";
```

Comment:
>c_defaultResourceFilename [](http://example.com/codeflow?start=18&length=25)

`c_defaultResourceIndexFilename` or `c_defaultPriFilename` would be better IMO since resource file can mean resw files, image files, etc.